backend/user-service/src/routes/user.routes.ts

```diff
@@ -1,8 +1,10 @@
+import { handleCreateUser, handleUpdateProfile } from '../controllers/user.controller'
+
 import { Router } from 'express'
-import { handleCreateUser } from '../controllers/user.controller'
 
 const router = Router()
 
-router.post('/', handleCreateUser)
+router.post('/create', handleCreateUser)
+router.post('/update-profile', handleUpdateProfile)
```

Comment:
You can use a PUT to update the profile. Also, regarding RESTFUL conventions, a POST / and PUT / is enough to represent a creation of a new resource and update of a new resource

Reply:
yeah true, just wanted to make it as clear as possible...  will make the change